app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -515,6 +517,7 @@ private fun ConversationScreen(
 
         ConversationScreenState.BottomSheetMenuType.None -> emptyList()
     }
+    // only here we will use normal Scaffold because of specific behaviour of message composer
     Scaffold(
```

Comment:
question: how does it differ from the `WireScaffold`? Is it only about `imePadding` not being added to the whole `Scaffold`? If so then can't we use the same `WireScaffold` everywhere and just provide a parameter to determine its behaviour? 🤔

Reply:
I would leave it as it is because someone may change something and may not know that it breaks Conversation screen